### ISSUE TITLE
Feature Request: disable automatic targeting of light sources for griefing

### ISSUE THREAD

**Issue by Sukasa** [Funwayguy/Epic-Siege-Mod#135]
Would be nice if we can disable automatic griefing of light sources and/or disable mob griefing entirely if we only want other features of the mod.

**Comment by BruhIndeed** [comment]
You can disable those in the config...

**Comment by Sukasa** [comment]
No, that allows you to add extra griefing targets, but any block that emits light is hard-coded in the AI from what I saw in the code
